compatibility/history-proxy/src/index.js

```diff
@@ -0,0 +1,63 @@
+const { Logger } = require('@dojot/microservice-sdk');
+const express = require('express');
+const axios = require('axios');
+const proxyMiddleware = require('./proxy');
+const configBase = require('./config');
+
+Logger.setTransport('console', {
+  level: 'debug',
+});
+Logger.setVerbose(true);
+const logger = new Logger('gui-proxy');
+
+const { port } = configBase;
+
+const app = express();
+
+app.get('/history/device/:deviceId/history', async (req, res) => {
```

Comment:
Maybe you can use web utils from sdk
https://github.com/dojot/dojot-microservice-sdk-js/blob/master/examples/webUtils/README.md

Reply:
Thanks, I'll check it.